src/Term/ItemTermStoreWriter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Term;
+
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Term\Fingerprint;
+
+interface ItemTermStoreWriter {
```

Comment:
BTW. This is going to replace EntityTermStoreWriter interface

Reply:
`EntityTermStoreWriter` is generally defined on EntityId/ EntityDocument, it might have a use for other entity types.